fsw/src/mm_app.c

```diff
@@ -420,11 +420,12 @@ bool MM_ResetCmd(const CFE_SB_Buffer_t *BufPtr)
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 bool MM_LookupSymbolCmd(const CFE_SB_Buffer_t *BufPtr)
 {
-    int32              OS_Status = OS_ERROR; /* Set to error instead of success since we explicitly test for success */
-    cpuaddr            ResolvedAddr   = 0;
-    MM_LookupSymCmd_t *CmdPtr         = NULL;
-    size_t             ExpectedLength = sizeof(MM_LookupSymCmd_t);
-    bool               Result         = false;
+    int32   OS_Status    = OS_ERROR; /* Set to error instead of success since we explicitly test for success */
+    cpuaddr ResolvedAddr = 0;
+    const MM_LookupSymCmd_t *CmdPtr         = NULL;
+    size_t                   ExpectedLength = sizeof(MM_LookupSymCmd_t);
+    bool                     Result         = false;
+    char                     SymName[OS_MAX_PATH_LEN];
```

Comment:
This should probably be `OS_MAX_SYM_LEN` rather than `OS_MAX_PATH_LEN`

Reply:
Thanks! Corrected